locales/it.json

```
@@ -0,0 +1,19 @@
+{
+    "status": {
+        "open": "aperto durante il blocco",
+        "closed": "chiuso durante il blocco"
+    },
+    "note": {
+        "header": "Npte #covid19 #iorestoaperto {HASHTAG_COUNTRY}",
+        "name": "Nome",
+        "status": "Stato",
+        "details": "Dettagli",
+        "url": "URL",
+        "footer": "Per chiudere questa note, usa opening_hours:covid19 e description:covid19 : https://wiki.openstreetmap.org/wiki/Key:opening_hours:covid19",
```

```suggestion
        "footer": "Per chiudere questa note, usa opening_hours:covid19 e description:covid19 : https://wiki.openstreetmap.org/wiki/IT:Key:opening_hours:covid19",
```

Did you plan on translating https://wiki.openstreetmap.org/wiki/Key:opening_hours:covid19 to Italian?

---

Yes, thanks!

---

Regarding the wiki, so far no time to do it, I'd leave English as default now and come back here and change when Italian version is ready

---

OK, I merge it now and feel free to edit on Transifex, as translations will be managed there from now.